code/modules/events/immovable_rod.dm

```diff
@@ -60,7 +63,10 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 			clong.ex_act(2)
 
 	else if (istype(clong, /mob))
+		if(istype(clong, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = clong
+			H << "The rod penetrates you!"

```

Comment:
Yes, it is a feedback message and the user is in extreme danger.
1. feedback messages relating to damage should not be in pure black (no spans)
2. feedback messages relating to the user taking damage or being in danger should use ```<span class='userdanger'>``` around the text.